delivery_recipe/delivery_team.md

```diff
@@ -0,0 +1,80 @@
+# Delivery Team
+
+This is a joint effort. Convivio don’t parachute into projects, do their magic then leave, like the Milk Tray Man. The best results are those delivered by a joint Convivio-client team.
+
+## Understanding the role of the team
+
+The ideal team is made up of experience, skill and authority. Experience provides us with deep, wide-ranging knowledge that we use to understand what needs to be done. The skill provides us with the tools that enable us to deliver great products. The authority clears the path of obstructions and allows “the team” to make decisions in the best interest of the business and its users.
```

Comment:
maybe referring to it as skills throughout would be clearer for what is intended here?